packages/components/src/color-palette/test/index.tsx

```diff
@@ -236,24 +239,25 @@ describe( 'ColorPalette', () => {
 		);
 
 		// Confirm the correct color name, color value, and button label are used
-		expect( screen.getByText( EXAMPLE_COLORS[ 0 ].name ) ).toBeVisible();
-		expect( screen.getByText( EXAMPLE_COLORS[ 0 ].color ) ).toBeVisible();
+		expect(
+			screen.getByText( colorName, { selector: 'span' } )
+		).toBeVisible();
+		expect( screen.getByText( colorCode ) ).toBeVisible();
 		expect(
 			screen.getByRole( 'button', {
-				name: `Custom color picker. The currently selected color is called "${ EXAMPLE_COLORS[ 0 ].name }" and has a value of "${ EXAMPLE_COLORS[ 0 ].color }".`,
+				name: `Custom color picker. The currently selected color is called "${ colorName }" and has a value of "${ EXAMPLE_COLORS[ 0 ].color }".`,
 				expanded: false,
 			} )
 		).toBeInTheDocument();
 
 		// Clear the color, confirm that the relative values are cleared/updated.
 		await user.click( screen.getByRole( 'button', { name: 'Clear' } ) );
 		expect( screen.getByText( 'No color selected' ) ).toBeVisible();
+
 		expect(
-			screen.queryByText( EXAMPLE_COLORS[ 0 ].name )
-		).not.toBeInTheDocument();
-		expect(
-			screen.queryByText( EXAMPLE_COLORS[ 0 ].color )
+			screen.queryByText( colorName, { selector: 'span' } )
```

Comment:
This selector could be made further specific like we did for the color code and color name queries above, but for non-existence queries it should be fine as-is.

Reply:
I missed this earlier, but I came across it when I updated the selector to class names, so this should be set with the latest commit. 😄